src/components/Auth/Auth.tsx

```diff
@@ -2,7 +2,7 @@ import React, { Component, Fragment } from 'react'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import { ForgotPasswordModal, LoginModal, SignUpModal } from 'podverse-ui'
-import { internetConnectivityErrorMessage } from '~/lib/constants/misc'
+import { internetConnectivityErrorMessage, constants } from '~/lib/constants/misc'
```

Comment:
When destructuring imported variables, please put them in alphabetical order.

This is actually something a linter can automatically fix for us, but we had to disable the linter on your system because of the husky issue we ran into :/